tests/testsuite.src/used_binaries.at

```diff
@@ -1104,7 +1104,7 @@ AT_CHECK([$COMPILE_MODULE prog.cob], [0], [], [])
 
 # mismatch in function signature - we ignore the error output,
 # as it depends on the C compiler in use
-AT_CHECK([$COMPILE_MODULE --include $(_return_path "$(pwd)/filec.h") -fstatic-call prog.cob], [1], [], [ignore])
+AT_CHECK([$COMPILE_MODULE --include $(_return_path "$(pwd)/filec.h") -fstatic-call prog.cob], [1], [ignore], [ignore])
```

Comment:
Under MSVC, the error seems to be displayed on stdout instead of stderr...

Reply:
Did you verify that with plain use of MSVC? I _guess_ this may be rooted in our process_filtered function which is only active under MSVC to filter the dumb output of the fiile that is processed.

Reply:
Same by calling `cl` directly:
```
C:\Users\David>cl /nologo test.c 1>out.txt 2>err.txt

C:\Users\David>type out.txt
test.c
test.c(11): error C2198: 'f' : pas assez d'arguments pour un appel

C:\Users\David>type err.txt

C:\Users\David>
```

Reply:
Thanks for checking - I've adjusted the testsuite.